oe_bootstrap_theme.theme

```diff
@@ -56,3 +56,29 @@ function oe_bootstrap_theme_preprocess_table__bootstrap__responsive(&$variables)
 
   $variables['wrapper_attributes'] = new Attribute(['class' => $class]);
 }
+
+/**
+ * Implements hook_preprocess_HOOK() for views-view-table.html.twig.
+ */
+function oe_bootstrap_theme_preprocess_views_view_table(&$variables) {
```

Comment:
Let's go with the same approach as we did with the table templates:

- implement `oe_bootstrap_theme_theme_suggestions_views_view_table_alter`
- there we add `views_view_table__bootstrap` and `views_view_table__bootstrap__responsive`.
- then  implement `oe_bootstrap_theme_preprocess_views_view_table__bootstrap__responsive`
- template `views-view-table--bootstrap.html.twig`
- and `views-view-table--bootstrap--responsive.html.twig` extending the first one